scripts/parser-tests/test262/allowlist.txt

```diff
@@ -1 +1,4 @@
-
+language/expressions/class/cpn-class-expr-fields-methods-computed-property-name-from-await-expression.js(default)
+language/expressions/class/cpn-class-expr-fields-methods-computed-property-name-from-await-expression.js(strict mode)
+language/statements/class/cpn-class-decl-fields-methods-computed-property-name-from-await-expression.js(default)
+language/statements/class/cpn-class-decl-fields-methods-computed-property-name-from-await-expression.js(strict mode)
```

Comment:
This is indeed a bug of `@babel/parser`

```
class C { [await 9] = () => {} }
```
should not throw
```
Can not use 'await' as identifier inside an async function (2:3)
```
when `sourceType` is `"module"` and `"topLevelAwait"` is enabled.